.eslintrc.js

```diff
@@ -43,13 +37,36 @@ module.exports = {
         '@typescript-eslint/no-object-literal-type-assertion': 'off',
         '@typescript-eslint/no-use-before-define': 'off',
         '@typescript-eslint/explicit-module-boundary-types': 'off',
-        '@typescript-eslint/no-duplicate-imports': 'error',
+        '@typescript-eslint/no-duplicate-imports': 'warn',
```

Comment:
Let it be an error. Warnings are usually used when ESLint is attached to old code base, and it is painful to change everything.